scripts/benchmarks/utils/utils.py

```diff
@@ -42,6 +42,11 @@ def run(command, env_vars={}, cwd=None, add_sycl=False):
 
 def git_clone(dir, name, repo, commit):
     repo_path = os.path.join(dir, name)
+    if not options.rebuild:
```

Comment:
no, please remove this. The intention is for the benchmark git repos to still be pulled to check for updates.

Reply:
Got it. Removed all conditions checking options.rebuild, which are unnecessary with this approach.